sentry-core/src/test/java/io/sentry/core/transport/HttpTransportTest.kt

```diff
@@ -297,6 +297,66 @@ class HttpTransportTest {
         assertTrue(transport.isRetryAfter("event"))
     }
 
+    @Test
+    fun `parse X-Sentry-Rate-Limit and ignore unknown categories`() {
+        val transport = fixture.getSUT()
+
+        whenever(fixture.connection.inputStream).thenThrow(IOException())
+        whenever(fixture.connection.getHeaderField(eq("X-Sentry-Rate-Limits")))
+            .thenReturn("60:default;wtf;error;security:organization")
+
+        val event = SentryEvent()
+
+        transport.send(event)
+        Thread.sleep(2000)
+        assertFalse(transport.isRetryAfter("wtf"))
```

Comment:
`m`: Maybe it makes sense to assert that the other categories have a retry after active.
Furthermore, we are going away from `Retry-After` to `Rate-Limits`, maybe it makes sense to rename this method at some point, it's a little bit confusing for me.

Reply:
renaming would be a breaking change as Android is released like this since 2.0 GA.
but still it's a `retry after` approach. why would it not make sense?

Reply:
Because it's our custom rate limit approach and I think we just used the retry after header to implement rate-limiting in the first place. But it's not a big deal.